helm_drydock/ingester/plugins/yaml.py

```diff
@@ -91,19 +79,275 @@ def parse_docs(self, yaml_string):
         for d in parsed_data:
             kind = d.get('kind', '')
             if kind != '':
-                if kind in YamlIngester.kind_map:
-                    try:
-                        model = YamlIngester.kind_map[kind](**d)
+                if kind == 'Region':
```

Comment:
Shouldn't there be a generic processor that handles most fields, like apiVersion, name, and so on?